plugin/trace/trace.go

```diff
@@ -26,6 +26,8 @@ const (
 	tagName  = "coredns.io/name"
 	tagType  = "coredns.io/type"
 	tagRcode = "coredns.io/rcode"
+	tagProto = "coredns.io/proto"
+	tagClientIP = "coredns.io/client-ip"
```

Comment:
to match the log plugin and others this should probably be renamed to:

```suggestion
	tagRemote = "coredns.io/remote"
```